difflint/lint.py

```diff
@@ -41,44 +41,30 @@ def run_command(self, args):
         self.output += output_bytes.decode()
 
 
-def _lint_javascript(file_to_lint, mode=None):
+def _lint_javascript(file_to_lint):
```

Comment:
Yes; a `.jshintrc` or `.jscsrc` applies recursively to all files in that directory or below. So if you have a directory with node-style Javascript and a directory with web-style Javascript in one repository, put separate configuration files in both of those directories.